tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -126,8 +125,7 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
 
                 HttpRequest httpRequest = httpContext.Request;
                 SpanContext propagatedContext = null;
-                var tagsFromHeaders = Enumerable.Empty<KeyValuePair<string, string>>();
-
+                var tagsFromHeaders = (IEnumerable<KeyValuePair<string, string>>)Array.Empty<KeyValuePair<string, string>>();
```

Comment:
I'd be interested if this _actually_ makes any difference, given it's a lot uglier 😛 And it's implemented as:

```csharp
public static IEnumerable<TResult> Empty<TResult>() => EmptyEnumerable<TResult>.Instance;

internal class EmptyEnumerable
{
    public static readonly TElement[] Instance = new TElement[0];
}
```

Reply:
changed in [9b271d6](https://github.com/DataDog/dd-trace-dotnet/pull/3735/commits/9b271d6330489f7f37df13159a5f41abe6ef8bb8)